blocks.js

```diff
@@ -45,44 +45,52 @@ var config = {
 	],
 	blocks: [
 		{
+			id: 'paragraph',
 			label: 'Paragraph',
 			icon: '<svg height="24" width="24" class="type-icon-paragraph" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path id="path-1_2_" class="st0" d="M13 5h2v16h2V5h2V3h-6.7.2-3C6.5 3 4 5.5 4 8.5S6.5 14 9.5 14H11v7h2v-7h-.5.5V5z"/><path class="st1" d="M9.5 3C6.5 3 4 5.5 4 8.5S6.5 14 9.5 14H11v7h2V5h2v16h2V5h2V3H9.5z"/></svg>',
-			categories: [ 'frequent' ]
+			category: 'frequent'
```

Comment:
Couldn't some blocks appear in multiple categories?  I'm guessing that was the initial intention of having an array here.